<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1589</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EndoTrack</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <widget class="QToolButton" name="zoomInPushButton">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="zoomOutPushButton">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="zoomMaximizePushButton">
        <property name="text">
         <string>^</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="separateLiveViewToolButton">
        <property name="text">
         <string>v</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="saveImageButton">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="settingsPushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Parameters</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_paintArea1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="PaintArea" name="paintArea1">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>512</width>
            <height>512</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>CrossCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_paintArea2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="PaintArea" name="paintArea2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>512</width>
            <height>512</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>CrossCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_controls" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
           <item>
            <widget class="QToolButton" name="playButton">
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="stopButton">
             <property name="text">
              <string>!</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="prevButton">
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="nextButton">
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="bookPositionSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="bookPositionLabel">
             <property name="text">
              <string>0/0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1589</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidget_parameters">
   <property name="minimumSize">
    <size>
     <width>262</width>
     <height>406</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Parameters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="parametersDockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="ParametersToolBox" name="toolBox_Parameters" native="true">
       <property name="currentIndex" stdset="0">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveParametersPushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restoreDefaultsPushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Restore defaults</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_actions">
   <property name="minimumSize">
    <size>
     <width>285</width>
     <height>796</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Actions</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="actionsDockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QStackedWidget" name="actionWidgetStack"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintArea</class>
   <extends>QLabel</extends>
   <header>PaintArea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ParametersToolBox</class>
   <extends>QWidget</extends>
   <header>ParametersToolbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
